feature-group-definitions/calc.dist.yml

```diff
@@ -0,0 +1,24 @@
+# Generated from: calc.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: calc()
+description: The `calc()` CSS function computes mathematical expressions such a `calc(100%/3 - 1em)`.
+spec: https://drafts.csswg.org/css-values-3/#calc-notation
+caniuse: calc
+status:
+  baseline: high
+  baseline_low_date: 2017-10-17
```

Comment:
It looks like nested calc support increases the date: https://developer.mozilla.org/en-US/docs/Web/CSS/calc#browser_compatibility

We should probably use an override here.